frontend/tampere/enCustomizations.tsx

```diff
@@ -474,133 +474,136 @@ const en: DeepPartial<Translations> = {
-      <H2>Vaihtoehtoiset asiointitavat</H2>
+      <H2>Alternative ways of accessing the service</H2>
       <P>
-        <strong>Varhaiskasvatuksen ja esiopetuksen asiakaspalvelu</strong>
+        <strong>
+          Early childhood and pre-primary education customer service
+        </strong>
         <br />
-        sähköposti{' '}
+        email{' '}
         <a href="mailto:varhaiskasvatus.asiakaspalvelu@tampere.fi">
           varhaiskasvatus.asiakaspalvelu@tampere.fi
         </a>
         <br />
-        puhelin <a href="tel:040 800 7260">040 800 7260</a>, soittoaika ma–pe
-        klo 9.00–12.00
+        telephone <a href="tel:040 800 7260">040 800 7260</a>, calling hours
+        Mon-Fri 9.00–12.00
       </P>
-      <H2>Anna palautetta</H2>
+      <H2>Give feedback</H2>
       <P>
-        Jos huomaat saavutettavuuspuutteen verkkopalvelussamme, kerro siitä
-        meille.{' '}
+        If you notice an accessibility gap in our online service, please let us
+        know! You can give us feedback using the{' '}
         <ExternalLink
           href="https://elomake.tampere.fi/lomakkeet/21932/lomakkeet.html"
-          text="Anna saavutettavuuspalautetta tällä verkkolomakkeella"
+          text="Give accessibility feedback with this web form"
         />
       </P>
-      <H2>Valvontaviranomainen</H2>
+      <H2>Supervisory authority</H2>
       <P>
-        Jos huomaat sivustolla saavutettavuusongelmia, anna ensin palautetta
-        meille sivuston ylläpitäjille. Vastauksessa voi mennä 14 päivää. Jos et
-        ole tyytyväinen saamaasi vastaukseen, tai et saa vastausta lainkaan
-        kahden viikon aikana, voit antaa palautteen Etelä-Suomen
-        aluehallintovirastoon. Etelä-Suomen aluehallintoviraston sivulla
-        kerrotaan tarkasti, miten valituksen voi tehdä, ja miten asia
-        käsitellään.
+        If you notice any accessibility issues on the website, please send us,
+        the site administrator, feedback first. It may take us up to 14 days to
+        reply. If you are not satisfied with the reply or you do not receive a
+        reply within two weeks, you can give feedback to the Regional State
+        Administrative Agency for Southern Finland. The website of the Regional
+        State Administrative Agency for Southern Finland explains in detail how
+        a complaint can be submitted, and how the matter will be processed.
       </P>
 
       <P>
-        <strong>Valvontaviranomaisen yhteystiedot </strong>
+        <strong>Contact information of the supervisory authority</strong>
         <br />
-        Etelä-Suomen aluehallintovirasto <br />
-        Saavutettavuuden valvonnan yksikkö
+        Regional State Administrative Agency of Southern Finland
+        <br />
+        Accessibility Supervision Unit
         <br />
         <ExternalLink
-          href="www.saavutettavuusvaatimukset.fi"
-          text="www.saavutettavuusvaatimukset.fi"
+          href="https://www.webaccessibility.fi/"
```

Comment:
Espoo links the Finnish web page, I used the English version (the URL that loads when you select English from the language menu of the original site) instead.